packages/SidePanel/src/components/Footer/Footer.styles.js

```diff
@@ -1,10 +1,32 @@
-import { css } from "styled-components";
+import { css, keyframes } from "styled-components";
 import stylers from "@paprika/stylers";
 import tokens from "@paprika/tokens/lib/tokens";
 
+function slideIn() {
+  return keyframes`
+    0% { width: 100%; }
+    100% { width: 100%; }
+  `;
+}
+
+function slideOut(width) {
+  return keyframes`
+    0% { width: 100%; }
+    99% { width: 100%; }
+    100% { width: ${width}; }
+  `;
+}
+
 const fixedFooterStyles = css`
-  position: fixed;
-  width: ${props => props.width};
+  ${props => {
+    const animation = props.isOpen ? slideIn() : slideOut(props.width);
```

Comment:
nit: deconstruct `props.width` to a variable as it is used twice? looks a bit cleaner imo